packages/flutter/lib/src/material/banner.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'banner_theme.dart';
+import 'button_bar.dart';
+import 'button_theme.dart';
+import 'divider.dart';
+import 'theme.dart';
+
+/// A Material Design banner.
+///
+/// A banner displays an important, succinct message, and provides actions for
+/// users to address (or dismiss the banner). It requires a user action to be
+/// dismissed.
```

Comment:
A user action is required for it to be dismissed.

Reply:
done